quinn/extensions/spark_session_ext.py

```diff
@@ -1,48 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one or more
-# contributor license agreements.  See the NOTICE file distributed with
-# this work for additional information regarding copyright ownership.
-# The ASF licenses this file to You under the Apache License, Version 2.0
-# (the "License"); you may not use this file except in compliance with
-# the License.  You may obtain a copy of the License at
-# http://www.apache.org/licenses/LICENSE-2.0
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-from __future__ import annotations
-
-import warnings
-
-from pyspark.sql import DataFrame, SparkSession
-from pyspark.sql.types import StructField, StructType
-
-
-def create_df(
```

Comment:
I see it like we do not delete this function, but delete the patching (row 48)

Reply:
Hello @SemyonSinchenko . Thanks for your review & suggestion. I have deleted the patching i.e., row 48 from this line instead of the whole file.

I have also added back the `extensions/__init__.py` with the reference to the `spark_session_ext`.

Also the doc-string didn't align with the parameters/arguments passed to the function. So, I have fixed that too. [Here's the link to the commit](https://github.com/MrPowers/quinn/pull/239/commits/483b33481e000aac1b301b020a71b9cb9dd3ef09).

Let me know your thoughts on the change?

Reply:
@nijanthanvijayakumar What do you think about moving this `create_dataframe` function to `quinn/dataframe_helpers.py`? And delete the extensions file